src/data/faq.json

```diff
@@ -87,6 +87,117 @@
                     }
                 ]
             },
+            {
+                "title": "Schnelltest",
+                "id": "rapid_antigen_test",
+                "active": true,
+                "accordion": [
+                    {
+                        "title": "Why are quick tests now integrated?",
+                        "anchor": "rat_why_integrate_rat",
+                        "active": true,
+                        "textblock": [
+                            "In the latest version of the Corona-Warn-App (2.1), the project team from Deutsche Telekom and SAP has integrated the announced rapid tests so that chains of infection can be interrupted even faster.",
+                            "While PCR test results are only available and can be shared a day later, users can share positive rapid test results shortly after the test. This means that contacts are warned more quickly and chains of infection are broken more quickly.",
+                            "A negative test result can also be used by users as evidence of a negative rapid test result, if required by law. However, the recognition of test evidence may vary from state to state. Users should therefore inform themselves about the respective criteria in their federal state."
+                        ]
+                    },
+                    {
+                        "title": "How do I get my rapid antigen test result?",
+                        "anchor": "rat_how_get_result",
+                        "active": true,
+                        "textblock": [
+                            "From the 'Register your Test' section on the home screen of your Corona Warn App, you can add a test by scanning the QR code. To get the result in the app as soon as possible, you can refresh the app until the result is available. Although the Corona Warn App also retrieves test results automatically in the background, it may take a little longer depending on your operating system."

```

Comment:
```suggestion
                            "From the 'Register your Test' section on the home screen of your Corona-Warn-App, you can add a test by scanning the QR code. To get the result in the app as soon as possible, you can refresh the test in the app until the result is available. Although the Corona-Warn-App also retrieves test results automatically in the background, it may take a little longer depending on your operating system."

```